proto/storage/cve.proto

```diff
@@ -128,6 +136,8 @@ message ImageCVE {
   bool snoozed = 7 [deprecated = true]; // @gotags: search:"CVE Snoozed"
   google.protobuf.Timestamp snooze_start = 8 [deprecated = true];
   google.protobuf.Timestamp snooze_expiry = 9 [deprecated = true]; // @gotags: search:"CVE Snooze Expiry,hidden"
+  float nvdcvss = 10; // @gotags: search:"NVD CVSS",store"
```

Comment:
what score is this?  v2?  v3?  luckily I know very little about this stuff so I'm asking lots of questions based on my lack of knowledge

Reply:
this is cvss score from NVD. This field is added so that graphql resolvers can return top nvd scvss score for Workload CVEs table in UI

Reply:
this is cvss score returned by nvd

Reply:
So is this field something we would have in the data model in an ideal world OR is this something we have to have in the data model due to how graphql and the search framework function?

Just looking at these items, I find them to be a little confusing.  Some comments might help.

Reply:
@dashrews78  this should be in data model since this table is populated by central based on EmbeddedVulnerability

Reply:
not exactly what I was getting at.  Is this the only place this piece of data resides or is it buried in other objects as well?

Reply:
CVSS of various flavors appear numerous times in these objects

Reply:
@dashrews78 for this feat we will show nvd cvss for  image cves only

Reply:
Coming out of the meeting 9/4/2024, I thought we were going to put a comment here to better explain what this is.